src/pages/campaign/CSVUploader.tsx

```
@@ -0,0 +1,160 @@
+
+    const droppedFile = e.dataTransfer.files?.[0];
+    if (droppedFile) {
+      await processFile(droppedFile);
+    }
+  };
+
+  const clearFile = () => {
+    setError(null);
+    setUploadSuccess(false);
+    setRecipientCount(0);
+    onUpload(null as any, 0);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)